# HG changeset patch
# User david@evans-2.local
# Date 1218808400 18000
# Branch fix_parenthesis
# Node ID a4ddabf50a0f9adcc8ec5c9ef5d2d96eed24a8eb
# Parent  94b327b81f1653d9610b792351fb01854c8bedcf
Fix missing opening parenthesis.

diff -r 94b327b81f16 -r a4ddabf50a0f Cython/Compiler/ModuleNode.py
--- a/Cython/Compiler/ModuleNode.py	Fri Aug 15 15:04:23 2008 +0200
+++ b/Cython/Compiler/ModuleNode.py	Fri Aug 15 08:53:20 2008 -0500
@@ -422,7 +422,7 @@
         code.putln("#if PY_VERSION_HEX < 0x02060000")
         code.putln("  #define Py_REFCNT(ob) (((PyObject*)(ob))->ob_refcnt)")
         code.putln("  #define Py_TYPE(ob)   (((PyObject*)(ob))->ob_type)")
-        code.putln("  #define Py_SIZE(ob)   ((PyVarObject*)(ob))->ob_size)")
+        code.putln("  #define Py_SIZE(ob)   (((PyVarObject*)(ob))->ob_size)")
         code.putln("  #define PyVarObject_HEAD_INIT(type, size) \\")
         code.putln("          PyObject_HEAD_INIT(type) size,")
         code.putln("  #define PyType_Modified(t)")
